### PR TITLE
Format Adjustments

### PR DESCRIPTION
**Pull request by ghost**
Moved some examples of tags into a section separate from the description.

### CODE REVIEW

File: plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java [hunk 1]
```diff
@@ -57,7 +57,12 @@ public ObjectTag getObjectAttribute(Attribute attribute) {
         // @group properties
         // @description
         // Returns a copy of the item with the specified flag data action applied to it.
-        // For example: <[item].with_flag[myflagname]>, or <[item].with_flag[myflag:myvalue]>, or <[item].with_flag[mycounter:+:<[amount]>]>
+        // @example
+        // <[item].with_flag[myflagname]>
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java]: These are not valid/complete examples

File: plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java [hunk 2]
```diff
@@ -122,7 +128,12 @@ public void adjust(Mechanism mechanism) {
         // @input ObjectTag
         // @description
         // Modifies a flag on this item, using syntax similar to <@link command flag>.
-        // For example, 'flag:myflagname:!' will remove flag 'myflagname', or 'flag:myflagname:3' sets flag 'myflagname' to value '3'.
+        // @example
+        // Removes flag 'myflagname'
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java]: Needs a `# ` prefix

File: plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java [hunk 3]
```diff
@@ -122,7 +128,12 @@ public void adjust(Mechanism mechanism) {
         // @input ObjectTag
         // @description
         // Modifies a flag on this item, using syntax similar to <@link command flag>.
-        // For example, 'flag:myflagname:!' will remove flag 'myflagname', or 'flag:myflagname:3' sets flag 'myflagname' to value '3'.
+        // @example
+        // Removes flag 'myflagname'
+        // - flag:myflagname:!
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java]: this isn't valid usage of a mech at all? what is this

File: plugin/src/main/java/com/denizenscript/denizen/tags/core/TextTagBase.java [hunk 4]
```diff
@@ -36,8 +36,9 @@ public TextTagBase() {
         // @description
         // Returns a special chat code that makes the following text display the input hover text when the mouse is left over it.
         // This tag must be followed by an <&end_hover> tag.
-        // For example: - narrate "There is a <&hover[you found it!]>secret<&end_hover> in this message!"
         // Note that this is a magic Denizen tool - refer to <@link language Denizen Text Formatting>.
+        // @example
+        // - narrate "There is a <&hover[you found it!]>secret<&end_hover> in this message!"
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/tags/core/TextTagBase.java]: examples should have `# comments` on them noting what the usage does in practice

**Comment by mcmonkey4eva** [comment]
do not touch the resolve button please

File: plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java [hunk 5]
```diff
@@ -57,7 +57,15 @@ public ObjectTag getObjectAttribute(Attribute attribute) {
         // @group properties
         // @description
         // Returns a copy of the item with the specified flag data action applied to it.
-        // For example: <[item].with_flag[myflagname]>, or <[item].with_flag[myflag:myvalue]>, or <[item].with_flag[mycounter:+:<[amount]>]>
+        // @example
+        // # Shows your held item with the flag 'myflagname'
+        // <player.item_in_hand.with_flag[myflagname]>
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java]: these are still not well formed examples

File: plugin/src/main/java/com/denizenscript/denizen/tags/core/TextTagBase.java [hunk 6]
```diff
@@ -71,9 +75,11 @@ public TextTagBase() {
         // Returns a special chat code that makes the following text execute the input command line value when clicked.
         // To execute a command "/" should be used at the start. Otherwise, it will display as chat.
         // This tag must be followed by an <&end_click> tag.
-        // For example: - narrate "You can <&click[wow]>click here<&end_click> to say wow!"
-        // For example: - narrate "You can <&click[/help]>click here<&end_click> for help!"
         // Note that this is a magic Denizen tool - refer to <@link language Denizen Text Formatting>.
+        // @example
+        // - narrate "You can <&click[wow]>click here<&end_click> to say wow!"
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/tags/core/TextTagBase.java]: same as above, examples should have `# comments` on them noting what the usage does in practice

File: plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java [hunk 7]
```diff
@@ -122,7 +132,12 @@ public void adjust(Mechanism mechanism) {
         // @input ObjectTag
         // @description
         // Modifies a flag on this item, using syntax similar to <@link command flag>.
-        // For example, 'flag:myflagname:!' will remove flag 'myflagname', or 'flag:myflagname:3' sets flag 'myflagname' to value '3'.
+        // @example
+        // # Removes flag 'myflagname' from your held item
+        // - flag <player.item_in_hand> myflagname:!
+        // @example
+        // # Sets flag 'myflagname' on your held item to value '3'
+        // - flag <player.item_in_hand> myflagname:3
```
Comment (tal5) [inline plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java]: Don't think these are correct? this is the `flag` ItemTag mechanism, not the `flag` command

**Pull request closed without merging**